consensus/clique/clique.go

```diff
@@ -324,6 +324,12 @@ func (c *Clique) verifyCascadingFields(chain consensus.ChainHeaderReader, header
 	if parent.Time+c.config.Period > header.Time {
 		return errInvalidTimestamp
 	}
+	// Verify the header's EIP-1559 attributes.
+	if chain.Config().IsLondon(header.Number) {
+		if err := misc.VerifyEip1559Header(chain.Config(), parent, header); err != nil {
+			return err
+		}
+	}
```

Comment:
Need to also have an else-block to verify that the header does not have a basefee. And I don't quite get why there's no corresponding `if header.GasUsed > header.GasLimit ` for clique (I mean previously) ?

Reply:
You're right, I'll add this.

Not sure why that check wasn't here previously though. Should I add it?